### DISCUSSION TITLE
OpenDX vs DXVK

### DISCUSSION THREAD

**Discussion by dimseal** [EduApps-CDG/OpenDX#12] [General]
Will the performance of OpenDX be greater than DXVK?

**Comment by EduApps-CDG** [comment]
Hey @dimseal!
DXVK is a Vulkan based implementation, which is uses the Linux Kernel's Direct Rendering Infrastructure. This means you have an extra abstraction layer.

OpenDX is currently in its initial stages, but I am dedicating my efforts to creating the fastest implementation possible despite the challenges I face, especially the difficulty in finding documentation for the DRM interface.

I already read how to create a software rendered buffer, but I still got nothing about hardware acceleration (which would make a difference).
